backend/mora/service/orgunit.py

```diff
@@ -45,6 +46,37 @@
                             url_prefix='/service')
 
 
+def _read_local_settings(unitid=None):
+    """ Read a set of settings from the database
+    :param query: The query
+    """
+    query_start = "SELECT setting, value FROM orgunit_settings WHERE object "
+    with sqlite3.connect(settings.USER_SETTINGS_DB_FILE,
+                         check_same_thread=False) as conn:
+        cur = conn.cursor()
+
+        if unitid is None:
+            query = query_start + "is Null"
+            cur.execute(query)
+        else:
+            query = query_start + "= ?"
+            cur.execute(query, (unitid,))
+
+        user_settings = {}
+
+        rows = cur.fetchall()
+        for row in rows:
+            setting = row[0]
+            if row[1] == 'True':
```

Comment:
Either factor this into a helper function, or — even better — use a database/library that retains types.